src/main/webapp/app/entities/programming-exercise.model.ts

```diff
@@ -17,6 +17,7 @@ export class ProgrammingExercise extends Exercise {
     public testRepositoryUrl: string;
     public publishBuildPlanUrl = false; // default value
     public allowOnlineEditor = false; // default value
+    public noLocalSetup = false; // default value
```

Comment:
please rename, move into section helper attributes and add a js doc comment what this is about

Reply:
I renamed and commented the checkbox name. I also adjusted the translation of the checkbox